Exiled.Events/Handlers/Player.cs

```diff
@@ -692,8 +694,10 @@ public static class Player
         /// <summary>
         /// Called after a <see cref="API.Features.Player"/> has spawned.
         /// </summary>
-        /// <param name="ev">The <see cref="SpawnedEventArgs"/> instance.</param>
-        public static void OnSpawned(SpawnedEventArgs ev) => Spawned.InvokeSafely(ev);
+        /// <param name="hub">The <see cref="ReferenceHub"/> instance.</param>
+        /// <param name="oldRole">The player's old <see cref="PlayerRoleBase"/> instance.</param>
+        /// <param name="newRole">The player's new <see cref="PlayerRoleBase"/> instance.</param>
+        public static void OnSpawned(ReferenceHub hub, PlayerRoleBase oldRole, PlayerRoleBase newRole) => Spawned.InvokeSafely(new SpawnedEventArgs(hub));
```

Comment:
We could add the "OldRole" property to `SpawnedEventArgs`.

Reply:
Yea, we can, but we cant add that property to Spawning event, i think it'll break logic

Reply:
Sure, add that property only to the `Spawned` event.

Reply:
Uh, ok